_includes/tools/collins_go_cli.html

```diff
@@ -0,0 +1,50 @@
+<p class="lead">Simple CLI access to Collins in Golang</p>
+<h2>Overview</h2>
+
+<p><code>collins-go-cli</code> allows you to interact with Collins from the command line. This includes changing attributes, provisioning machines, allocating IPs, maintenance-moding machines, etc. Download it from <a href="https://github.com/michaeljs1990/collins-go-cli/releases">collins-go-cli</a> on Github or if you have golang installed you can <code>go get -u cgit.xrt0x.com/xrt0x/collins-go-cli/cmd/collins</code>. The source can be found at <a href="https://github.com/michaeljs1990/collins-go-cli">github.com/michaeljs1990/collins-go-cli</a>.
+
+<h2>Installation</h2>
+<pre>
+$ go get -u cgit.xrt0x.com/xrt0x/collins-go-cli/cmd/collins
```

Comment:
porque no `github.com/tumblr/collins-go-cli`?

Reply:
ah so the package above it literally a cloned version of your collins-cli ruby gem ported to go with some additional features. I can add docs for the tumblr/go-collins library as well though.